Documentation/Deployment.md

```diff
@@ -9,9 +9,9 @@ Once content is checked into master, the template content is packaged into an np
 
 3. Every weekday @ 3pm (pacific), a daily build of the Application Insights Azure Portal extension takes place. This build consumes the NPM package from the ADO package feed.
 
-4. Every weekday @ ~4pm (pacific), a daily deployment deploys the extension to a pre-production environment (PPE)
+4. Every weekday @ ~4pm (pacific), [a daily deployment of the AppInsightsExtension occcurs](https://eng.ms/docs/cloud-ai-platform/azure/aep-platform-infrastructure/observability/application-insights/portal/operations/deployment#deployment-pipeline), which deploys the extension to a pre-production environment ([PPE](https://portal.azure.com/?feature.canmodifystamps=true&appInsightsExtension=ppe))
 
-5. Every weekday @ ~noon (pacific), the previous day's build moves up a stage, from PPE to another internal environment (MPAC), and other non-public cloud test environments.
+5. Every weekday @ ~noon (pacific), the previous day's build moves up a stage, from PPE to another internal environment ([MPAC](https://portal.azure.com/?feature.canmodifystamps=true&appInsightsExtension=mpac)), and other non-public cloud test environments.
```

Comment:
>internal environment [](http://example.com/codeflow?start=99&length=20)

do we need to point out that internal users will be automatically routed to mpac?

Reply:
I don't think we need to explain that detail here.  (it also is only true for some of the microsoft tenants, not all of them?)  this explictly loads that version whether you'd get autoredirected or not.